tendrl/commons/etcdobj/__init__.py

```diff
@@ -79,8 +79,21 @@ def save(self, obj):
         :rtype: EtcdObj
         """
         for item in obj.render():
-            LOG.debug("Writing %s to %s" % (item['key'], item['value']))
-            self.client.write(item['key'], item['value'], quorum=True)
+            try:
+                Event(
+                    Message(
+                        priority="debug",
+                        publisher=tendrl_ns.publisher_id,
+                        payload={"message": "Writing %s to %s" %
+                                            (item['key'], item['value'])
+                                 }
+                    )
+                )   
+                self.client.write(item['key'], item['value'], quorum=True)
+            except AttributeError:
```

Comment:
Why few places we are using try-except? I feel you should add some comments in this cases..